src/styles/SearchResults.css

```diff
@@ -0,0 +1,34 @@
+.search-results {
+  margin-top: 30px;
```

Comment:
`align-items` and `flex-direction` can both be removed from here as they arent doing anything